### PR TITLE
Updated examples in Section 5.6

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

File: draft-ietf-tls-dtls13.md [hunk 1]
```diff
@@ -1174,69 +1174,88 @@ is received. (See {{dtls-epoch}} for the definitions of each epoch.)
...
+|     |        |    x   | ServerHello, EncryptedExtensions, CertificateRequest*, Certificate*, CertificateVerify*, Finished | 
+| (2) |    x   |        | Certificate*, CertificateVerify*, Finished |
+|     |        |    x   | ACK | 
+| (3) |        |    x   | NewSessionTicket | 
+{: #tab-flights title="Flight Handshake Message Combinations."}
+
+Remarks: 
+
+- The HelloRetryRequest shown in (1) is an optional message sent by the server. 
+- The ACK message is sent by a client in response to a NewSessionTicket message from the server (as noted in (3)) or sent by a server in response to receipt of flight (2). In those cases it represent a flight on its own.
```
Comment (ekr) [inline draft-ietf-tls-dtls13.md:1194]: I tend to think we should not think of an ACK as part of a flight. It's kind of outside this model.

File: draft-ietf-tls-dtls13.md [hunk 2]
```diff
@@ -1174,69 +1174,88 @@ is received. (See {{dtls-epoch}} for the definitions of each epoch.)
...
+
+| Note | Client | Server | Handshake Messages |
+|-----+--------+--------+------------------ |
+|     |    x   |        | ClientHello |
+| (1) |        |    x   | HelloRetryRequest |
+|     |        |    x   | ServerHello, EncryptedExtensions, CertificateRequest*, Certificate*, CertificateVerify*, Finished | 
+| (2) |    x   |        | Certificate*, CertificateVerify*, Finished |
+|     |        |    x   | ACK | 
+| (3) |        |    x   | NewSessionTicket | 
+{: #tab-flights title="Flight Handshake Message Combinations."}
```
Comment (ekr) [inline draft-ietf-tls-dtls13.md:1189]: I don't quite see what the Note column is doing.

**Review by ekr — COMMENTED** [review]
@kaduk is this what you had in mind.

File: draft-ietf-tls-dtls13.md [hunk 3]
```diff
@@ -1174,69 +1174,88 @@ is received. (See {{dtls-epoch}} for the definitions of each epoch.)
...
+handshake message transmission of one peer and ends with the expected response from the 
+other peer. Table {{tab-flights}} contains a complete list of message combinations that consitute flights. 
+
+| Note | Client | Server | Handshake Messages |
+|-----+--------+--------+------------------ |
+|     |    x   |        | ClientHello |
+| (1) |        |    x   | HelloRetryRequest |
+|     |        |    x   | ServerHello, EncryptedExtensions, CertificateRequest*, Certificate*, CertificateVerify*, Finished | 
+| (2) |    x   |        | Certificate*, CertificateVerify*, Finished |
+|     |        |    x   | ACK |
```
Comment (kaduk) [inline draft-ietf-tls-dtls13.md:1187]: (A client will send ACK in response to post-handshake messages, just not during the initial handshake.  I think this is okay as the table contents, but maybe we should ensure that the prose is clear this is just for the initial handshake.)  Though, given ekr's comment below, perhaps it's going away anyway and becomes irrelevant.

File: draft-ietf-tls-dtls13.md [hunk 4]
```diff
@@ -1275,19 +1294,15 @@ Client                                            Server
...
+                                                          +--------+
+[ACK]                  -------->                          | Flight |
+                                                          +--------+
 ~~~
-{: #dtls-post-handshake-ticket title="Message flights for the new session ticket message"}
-
-Note: The application data sent by the client is not included in the
-timeout and retransmission calculation.
-
+{: #dtls-post-handshake-ticket title="Message flights for the NewSessionTicket message"}
```
Comment (kaduk) [inline draft-ietf-tls-dtls13.md:1305]: Do we also want to mention KeyUpdate/NewConnectionID/RequestConnectionID here?  It seems overkill to have a separate figure showing the message and ACK for each of them...

**Review by kaduk — COMMENTED** [review]
Thanks, this addresses the issues I had raised for this section.
(It's perhaps a little janky to have just a box with "Flight" for each one, but I don't have any better ideas, so we should probably roll with it!)

File: draft-ietf-tls-dtls13.md (hunk 3 above)
Comment (ekr) [reply draft-ietf-tls-dtls13.md:1187]: I actually left ACK in because it was more confusing not to.

File: draft-ietf-tls-dtls13.md (hunk 4 above)
Comment (ekr) [reply draft-ietf-tls-dtls13.md:1305]: Yeah, I guess that would work.

**Comment by ekr** [comment]
@kaduk I merged this. Any chance you want to send a PR or do you want me to?

**Comment by kaduk** [comment]
> @kaduk I merged this. Any chance you want to send a PR or do you want me to?

I'm not in a great spot to make a PR at the moment; please go ahead.

**Comment by ekr** [comment]
I just fixed this in the main branch.

**Pull request merged**